CHANGELOG.md

```diff
@@ -13,23 +13,26 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Include Card Separator example in Card kit docs ([#439][] @rruiz85)
 - Text Input Dark mode added, Text Input updated to use Textarea mixins and Textarea mixin fix to remove number field arrows ([#494][] @megantrimble)
 - Introduce Pull Request Template ([#508][] @rafbgarcia)
+- Extend Form Kit to use Textarea Kit for textarea form fields ([#506][] @roxannecojocariu)
 
 [#498]: https://github.com/powerhome/playbook/pull/498
 [#439]: https://github.com/powerhome/playbook/pull/439
+[#494]: https://github.com/powerhome/playbook/pull/494
 [#508]: https://github.com/powerhome/playbook/pull/508
+[#506]: https://github.com/powerhome/playbook/pull/506
 
-### Fixed
-
-- Display wrong numbers in Person Contact Kit. The Kit will now display a caption "wrong number" and have the wrong numbers listed beneath the caption ([#499][] @KFad11)
-
-[#499]: https://github.com/powerhome/playbook/pull/499
+### Changed
+- Adjusted margins for Textarea Kit to be consistent with TextInput Kit ([#506][] @roxannecojocariu)
 
+[#506]: https://github.com/powerhome/playbook/pull/506
 
 ### Fixed
 
 - Fixes master branch (regression introduced by myself on #498) ([#500][] @rafbgarcia)
+- Display wrong numbers in Person Contact Kit. The Kit will now display a caption "wrong number" and have the wrong numbers listed beneath the caption ([#499][] @KFad11)
```

Comment:
@KFad11 I noticed you created a new Fixed section in the same Unreleased section so I went ahead and added yours in the appropriate section (just in case there is any confusion).